engine/src/main/resources/assets/i18n/menu_uk.lang

```diff
@@ -115,10 +115,10 @@
     "video-preset": "Передустановка", 
     "video-preset-custom": "Користувацькі", 
     "video-preset-epic": "Епічні", 
-    "video-preset-insane": "Шалені",
+    "video-preset-insane": "Шалені", 
```

Comment:
@Cervator - Ukrainian looks good! Random whitespace change here, but that's probably just a Weblate formatting quirk.